src/tests/PostTest.test.ts

```diff
@@ -304,6 +305,60 @@ describe('post tests', () => {
     expect(getPostsResponse.posts).toEqual([]);
   });
 
+  test('filter posts by price', async () => {
+    const post = PostFactory.fakeTemplate();
+    post.user = UserFactory.fakeTemplate();
+
+    await new DataFactory()
+      .createPosts(post)
+      .createUsers(post.user)
+      .write();
+
+    expectedPost.user = post.user;
+
+    let filter = {
+      lowerBound: 1.0,
+      upperBound: 1000.0,
+    };
+
+    let getPostsResponse = await postController.filterPostsByPrice(filter);
+    getPostsResponse.posts[0].price = Number(getPostsResponse.posts[0].price);
+    expectedPost.created = getPostsResponse.posts[0].created;
+
+    expect(getPostsResponse.posts).toEqual([expectedPost]);
+
+    filter = {
+      lowerBound: 500.15,
+      upperBound: 1000.0,
+    };
+
+    getPostsResponse = await postController.filterPostsByPrice(filter);
+    getPostsResponse.posts[0].price = Number(getPostsResponse.posts[0].price);
+    expectedPost.created = getPostsResponse.posts[0].created;
+
+    expect(getPostsResponse.posts).toEqual([expectedPost]);
+
+    filter = {
+      lowerBound: 1.0,
+      upperBound: 500.15,
+    };
+
+    getPostsResponse = await postController.filterPostsByPrice(filter);
+    getPostsResponse.posts[0].price = Number(getPostsResponse.posts[0].price);
+    expectedPost.created = getPostsResponse.posts[0].created;
+
+    expect(getPostsResponse.posts).toEqual([expectedPost]);
+
+    filter = {
+      lowerBound: 1.0,
+      upperBound: 2.0,
+    };
+
+    getPostsResponse = await postController.filterPostsByPrice(filter);
```

Comment:
how are these test cases different from each other? if they're not, label this test case as something a little more informative (e.g. `filter posts by price, where price is within range`). if they are, separate them into different test cases.